src/DevBetterWeb.Web/Controllers/VideosController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using DevBetterWeb.Vimeo.Services.VideoServices;
+using DevBetterWeb.Web.Models.Vimeo;
+using Microsoft.AspNetCore.Mvc;
+
+
+namespace DevBetterWeb.Web.Controllers
+{
+  [Route("videos")]
+  [ApiController]
+  public class VideosController : Controller
+  {
+    private readonly IMapper _mapper;
+    private readonly GetAllVideosService _getAllVideosService;
```

Comment:
This is still calling out to the video provider, not using a local database, right? Why is this depending on an instance type, not an interface?